.changeset/long-buttons-burn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': patch
+---
+
+Allow IconButton to have a loading state
```

Comment:
Allowed the loading state for the `IconButton` component. Pass an `isLoading` prop with an accessible `loadingLabel` label.